.pre-commit-config.yaml

```diff
@@ -4,15 +4,14 @@ repos:
     hooks:
       - id: check-merge-conflict
   - repo: git://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.50.0
+    rev: v1.43.0
     hooks:
       - id: terraform_fmt
       - id: terraform_docs
-        args: ["--args=--sort-by required"]
```

Comment:
Shouldnt remove this.

Reply:
Having issues with this version so looked at other repos and they are using the following which works:

repos:
  - repo: git://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
      - id: check-merge-conflict
  - repo: git://github.com/antonbabenko/pre-commit-terraform
    rev: v1.43.0
    hooks:
      - id: terraform_fmt
      - id: terraform_docs
      - id: terraform_tfsec
      - id: terraform_tflint
  - repo: git://github.com/dnephin/pre-commit-golang
    rev: v0.3.5
    hooks:
      - id: go-fmt
      - id: go-lint

Reply:
Added back in:

 - id: terraform_docs
        args: ["--args=--sort-by required"]